apps/src/templates/projects/PersonalRecentProjects.jsx

```diff
@@ -32,17 +35,21 @@ const styles = {
 
 const ProjectCardGrid = React.createClass({
   propTypes: {
-    projectList: PropTypes.arrayOf(personalProjectDataPropType).isRequired
+    projectList: PropTypes.arrayOf(personalProjectDataPropType).isRequired,
+    isRtl: PropTypes.bool.isRequired
   },
 
   render() {
+    const { isRtl } = this.props;
+    const cardPaddingStyle = isRtl ? styles.cardPaddingRtl : styles.cardPadding;
```

Comment:
again, no need to make this a constant if it's only being used on the next line.